utils/api/definitions/schemas.ts

```diff
@@ -92,7 +92,7 @@ export const RequestSchema = Joi.object({
     .messages({ 'any.required': ValidationErrors.yearsSince18Empty })
     .integer()
     .max(Joi.ref('age', { adjust: (age) => age - 18 }))
-    .message(ValidationErrors.yearsInCanadaMinusAge),
+    .message(ValidationErrors.yearsSince18Empty),
```

Comment:
aren't you missing to change yearsInCanadaSinceOAS

Reply:
@MarcoGoC the changes are done, could you please review it and let me know :) 

Thanks!